src/snyk/common/vscode/nativeModules.ts

```diff
@@ -0,0 +1,44 @@
+export default [
```

Comment:
Please don't duplicate the code.

Reply:
I moved it to common, how is that a duplication?

Reply:
snyk/vscode-extension/src/snyk/snykOss/constants/nativeModules.ts